src/main/resources/js/creator/creator-utilities.js

```diff
@@ -227,6 +227,32 @@ function setCommentOnIssueUpdatesChoice(commentOnIssueUpdatesChoice) {
     }
 }
 
+function getProjectReviewerNotificationsChoice() {
+    const projectReviewerNotificationsChoiceElement = AJS.$("#" + "projectReviewerNotificationsChoice");
+
+    if (projectReviewerNotificationsChoiceElement[0].checked) {
+        return "true";
+    } else {
+        return "false";
+    }
```

Comment:
A thought, could we return something similar to projectReviewerNotificationsChoiceElement[0].checked.toString here instead of making a check?